src/vs/workbench/browser/parts/editor/breadcrumbsPicker.ts

```diff
@@ -129,13 +125,6 @@ export abstract class BreadcrumbsPicker {
 			this._layout();
 		}));
 
-		// filter on type: state
-		const cfgFilterOnType = BreadcrumbsConfig.FilterOnType.bindTo(this._configurationService);
-		this._tree.updateOptions({ filterOnType: cfgFilterOnType.getValue() });
```

Comment:
This overrides the setting `workbench.list.keyboardNavigation` and because `breadcrumbs.filterOnType` is commented its value is undefined/false, so the BreadcrumbsPicker tree won't filter on type after a launch/reload unless the TypeFilterController widget of the tree is clicked which modifes the `filterOnType` setting

Reply:
💡